docs: adding overview for operator installs (#156)
aerospike/kubernetes-operator: docs/Install-the-Operator-on-Kubernetes.md

This documentation includes examples of various Aerospike configuration settings and possibilities you can use with your Aerospike cluster on Kubernetes. If you have questions, suggestions, or other feedback, please let us know.

 
## Install the Operator

> There are two methods installing the Aerospike Kubernetes Operator itself. The primary, and recommended method, is to use the Operator Lifecycle Manager (OLM). The second method is to use Helm charts themselves to install the operator (which will then in turn manage the Aerospike clusters).

Reviewer: ```suggestion
There are two methods of installing the Aerospike Kubernetes Operator. The primary, and recommended method, is to use the Operator Lifecycle Manager (OLM). The second method is to use Helm charts to install the Operator, and then the Helm charts will manage the Aerospike clusters.
```

> If you are unsure which method to choose, we recommend using the OLM

Reviewer: ```suggestion
If you are unsure which method to choose, we recommend using the OLM.
```

### Method 1: Install the Operator Lifecycle Manager (OLM)

> We recommend using [Operator Lifecycle Manager (OLM)](https://olm.operatorframework.io/]) to run and manage the Aerospike Kubernetes Operator in production environments. OLM is the preferred way to manage Kubernetes operators in production.

Reviewer: ```suggestion
We recommend using [Operator Lifecycle Manager (OLM)](https://olm.operatorframework.io/]) to run and manage the Aerospike Kubernetes Operator in production environments. 
```

:::tip

> If you plan to use Helm charts to deploy Aerospike clusters, you will also need to [use Helm to install the Operator](install-operator-helm.md), instead.

Reviewer: ```suggestion
If you plan to use Helm charts to deploy Aerospike clusters, you must [use Helm to install the Operator](install-operator-helm.md).
```

:::

Follow the steps [here to install the Aerospike Kubernetes Operator using OLM](install-operator-olm)

### Method 2: Install the Operator using Helm Charts

> In some cases, particularly if you are planning on using helm charts to deploy Aerospike clusters directly (rather than relying on the operator), you may need to use the helm chart to also install the operator. 

Reviewer: ```suggestion
In some cases, particularly if you use Helm charts to deploy Aerospike clusters directly rather than relying on the Operator, you may need to use the Helm chart to also install the Operator. 
```

:::tip

> There are some mandatory, manual, steps if installing this way. Please ensure you need this behavior and follow the instructions closely.

Reviewer: ```suggestion
Follow the instructions closely as some manual steps are mandatory when you install the Operator using Helm charts. 
```

Author: we need to emphasize the manual steps I think
